DEV_SETUP.md

```diff
@@ -57,7 +55,7 @@ Save those backups to somewhere you'll be able to access from the new environmen
 Run the following commands:
 
     $ source /usr/local/bin/virtualenvwrapper.sh
-    $ mkvirtualenv --no-site-packages commcare-hq -p python3.6
+    $ python3.6 -m venv ~/.virtualenvs/commcare-hq
```

Comment:
`mkvirtualenv` is part of `virtualenvwrapper`, so can still be used, right? If not, then probably should also remove `virtualenvwrapper` above.
```suggestion
    $ mkvirtualenv --no-site-packages commcare-hq -p python3.6
```

Reply:
I like `virtualenvwrapper` because `workon` is great, and `setvirtualenvproject` is useful. ... But it seems `virtualenvwrapper` installs `virtualenv` as a requirement.

So unless we change our workflow to drop both `virtualenv` and `virtualenvwrapper` (maybe in favor of [direnv](https://direnv.net/)?) this PR is kind of pointless. I'm going to close it.